lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2426,9 +2426,10 @@ SwiftLanguageRuntime::GetRuntimeUnwindPlan(ProcessSP process_sp,
   // and thus it needs to be dereferenced to get to the context for the
   // function. The debug info for locals reflects this difference, so our
   // unwinding of the context reister needs to reflect it too.
-  bool in_resume_function =
-      sc.symbol->GetName().GetStringRef().contains(".resume.");
-  if (in_resume_function) {
+  bool indirect_context =
+      sc.symbol->GetName().GetStringRef().contains(
+          "await resume partial function");
+  if (indirect_context) {
```

Comment:
In other files we have helper functions to do this check using swift demangling. The same check would be to look for a `AsyncAwaitResumePartialFunction` top level child node. I don't think it blocks this, if you prefer to commit as is, it should be a relatively simple NFC later.

Note the comment above this still references ".resume.".